ui/components/component-library/avatar-token/avatar-token.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { SIZES } from '../../../helpers/constants/design-system';
+
+import README from './README.mdx';
+import { AvatarToken } from './avatar-token';
+
+export default {
+  title: 'Components/ComponentLibrary/AvatarToken',
+  id: __filename,
+  component: AvatarToken,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    size: {
+      control: 'select',
+      options: Object.values(SIZES),
+    },
+    tokenName: {
+      control: 'text',
+    },
+    tokenImageUrl: {
+      control: 'text',
+    },
+    showHalo: {
+      control: 'boolean',
+    },
+  },
+};
+
+export const DefaultStory = (args) => <AvatarToken {...args} />;
+
+DefaultStory.storyName = 'Default';
+
+export const AvatarTokenWithHalo = (args) => <AvatarToken {...args} />;
+
+AvatarTokenWithHalo.storyName = 'Halo';
+
+DefaultStory.args = {
+  tokenName: 'ast',
+  tokenImageUrl: './AST.png',
+  size: SIZES.MD,
+};
+
+AvatarTokenWithHalo.args = {
+  tokenName: 'ast',
+  tokenImageUrl: './AST.png',
+  size: SIZES.MD,
+  showHalo: true,
+};
```

Comment:
You can use the template style of writing stories if there are multiple stories that share the same structure. Also added some duplicate stories from BaseAvatar but I think it makes sense to. We probably want the engineers to be referencing base components as little as possible

```suggestion
  },
  args: {
    tokenName: 'ast',
    tokenImageUrl: './AST.png',
    size: SIZES.MD,
    showHalo: false,
  },
};

const Template = (args) => {
  return <AvatarToken {...args} />;
};

export const DefaultStory = Template.bind({});

DefaultStory.storyName = 'Default';

export const Size = (args) => (
  <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.BASELINE} gap={1}>
    <AvatarToken {...args} marginBottom={2} size={SIZES.XS} />
    <AvatarToken {...args} marginBottom={2} size={SIZES.SM} />
    <AvatarToken {...args} marginBottom={2} size={SIZES.MD} />
    <AvatarToken {...args} marginBottom={2} size={SIZES.LG} />
    <AvatarToken {...args} marginBottom={2} size={SIZES.XL} />
  </Box>
);

export const tokenName = Template.bind({});

tokenName.args = {
  tokenImageUrl: '',
};

export const tokenImageUrl = Template.bind({});

export const showHalo = Template.bind({});

showHalo.args = {
  showHalo: true,
};

export const BackgroundAndBorderColor = (args) => (
  <Box display={DISPLAY.FLEX} gap={1}>
    <AvatarToken
      {...args}
      backgroundColor={COLORS.KOVAN}
      borderColor={COLORS.KOVAN}
      tokenName="K"
      color={COLORS.PRIMARY_INVERSE} // This will have to be added to the Box component as a prop so we can change the color of the text and to the base avatar
    />
    <AvatarToken
      {...args}
      backgroundColor={COLORS.RINKEBY}
      borderColor={COLORS.RINKEBY}
      tokenName="R"
      color={COLORS.PRIMARY_INVERSE} // This will have to be added to the Box component as a prop so we can change the color of the text and to the base avatar
    />
    <AvatarToken
      {...args}
      backgroundColor={COLORS.GOERLI}
      borderColor={COLORS.GOERLI}
      tokenName="G"
      color={COLORS.PRIMARY_INVERSE} // This will have to be added to the Box component as a prop so we can change the color of the text and to the base avatar
    />
    <AvatarToken
      {...args}
      backgroundColor={COLORS.ROPSTEN}
      borderColor={COLORS.ROPSTEN}
      tokenName="R"
      color={COLORS.PRIMARY_INVERSE} // This will have to be added to the Box component as a prop so we can change the color of the text and to the base avatar
    />
  </Box>
);

BackgroundAndBorderColor.args = {
  tokenImageUrl: '',
};
```

Will need these imports for new stories

```
import {
  ALIGN_ITEMS,
  COLORS,
  DISPLAY,
  SIZES,
} from '../../../helpers/constants/design-system';
import Box from '../../ui/box/box';
```